bindings/python/quokka/program.py

```diff
@@ -440,6 +440,21 @@ def iter_chunk(
                 if chunk.chunk_type in chunk_types:
                     yield chunk
 
+    def read_byte(self, v_addr: AddressT, size: int) -> bytes:
```

Comment:
Can we call it ``read_bytes`` ? (otherwise it suggests we read only a single byte)

Reply:
I am also for using `read_bytes` but I wanted to keep consistency with `Executable.read_byte`. Maybe it's better to rename `Executable.read_bytes` as well, even though we are breaking backward compatibility